lively.modules/src/system.js

```diff
@@ -500,11 +619,11 @@ function printSystemConfig (System) {
   const json = {
     baseURL: System.baseURL,
     transpiler: System.transpiler,
-    defaultJSExtensions: System.defaultJSExtensions,
-    defaultExtension: System.defaultExtension,
+    defaultJSExtensions: System.defaultJSExtensions, // deprecated in 0.21
+    defaultExtension: System.defaultExtension, // wont work at all?? needs to be done via the package
```

Comment:
🇹🇴

Reply:
Pretty sure those can go, if they are not referenced anywhere outside of SystemJS.